examples/pymc3_howto/profiling.ipynb

```diff
@@ -31,16 +31,6 @@
     "print(f\"Running on PyMC3 v{pm.__version__}\")"
```

Comment:
<p>The parenthesis are used in the style guide to indicate that it's optional, many notebooks import theano and/or xarray and won't need that, they should be removed as I don't think it will work like this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/143/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>